go.mod

```diff
@@ -26,7 +25,10 @@ require (
 	golang.org/x/time v0.0.0-20220722155302-e5dcc9cfc0b9
 )
 
-require github.com/cenkalti/backoff v2.2.1+incompatible
+require (
+	github.com/cenkalti/backoff v2.2.1+incompatible
+	github.com/leovct/go-substrate-rpc-client/v4 v4.0.0-20230525153913-bfb420a865c0
```

Comment:
We reverted this changed so we currently use `github.com/centrifuge/go-substrate-rpc-client/v4`.

We can't use `github.com/availproject/avail/v4` there because the module name is [set to `centrifuge/go-substrate-rpc-client/v4`](https://github.com/availproject/go-substrate-rpc-client/blob/master/go.mod#L1) (the same as the original centrifuge repo) and not to something else like `avail/go-substrate-rpc-client/v4`. This means that avail load testing will probably not work from there. Nevertheless, it looks like this feature was broken since a long time.